src/site/layout/header.rs

```diff
@@ -53,11 +53,10 @@ fn nav(
         Message::new(MessageType::Info, "Found additional pages...").print();
         for (page_name, page_path) in additional_pages.iter() {
             if page::source::is_markdown(page_path) {
-                let file_path = page::source::get_filename(page_path);
+                let file_path = page::source::get_filename_with_dir(page_path);
 
                 if let Some(file_name) = file_path {
-                    let href =
-                        link::generate(path_prefix, &format!("{}/", file_name.to_string_lossy()));
+                    let href = link::generate(path_prefix, &format!("{}/", file_name));
```

Comment:
we should join paths with camino

Reply:
there's no path joining being done here, unless you mean the URL path, which is handled by the `link::generate` function, the `format!` is just for handling the dev-only trailing-slash quirk that we've talked about before

Reply:
ah yeah whoops, i forgot